README.md

```diff
@@ -1,179 +1,46 @@
-# 🚀 api
+<p align="center">
```

Comment:
This file is also the same as `packages/api/README.md` so you don't need to review both.